packages/office-ui-fabric-react/src/components/Button/CompoundButton/CompoundButton.styles.tsx

```diff
@@ -0,0 +1,73 @@
+import { concatStyleSets, FontWeights } from '../../../Styling';
+import { IButtonBaseStyleProps, IButtonBaseStyles } from '../_base/Button.base.types';
+
+import { getButtonBaseStyles } from '../_base/Button.base.styles';
+import { primaryStyles, standardStyles, primarySplitStyles, standardSplitStyles } from '../DefaultButton/ButtonThemes';
+
+export const getStyles = (props: IButtonBaseStyleProps): IButtonBaseStyles => {
+
+  const buttonStyles: IButtonBaseStyles = {
+    root: {
+      maxWidth: '280px',
+      height: 'auto',
+    },
+    button: {
+      minHeight: '72px',
```

Comment:
same comment about the number value for single value css properties in this and other files